collect_app/src/main/java/org/odk/collect/android/tasks/DownloadFormsTask.java

```diff
@@ -240,6 +252,19 @@ private void cleanUp(FileResult fileResult, File fileOnCancel, String tempMediaP
             Timber.w("The user cancelled (or an exception happened) the download of a form at the "
                     + "very beginning.");
         } else {
+            Cursor c = null;
```

Comment:
What do you think about moving this code elsewhere? This module is very long already. Maybe formsDao.deleteFormsFromMd5Hash? That would also make it clear at a glance what these 12 lines do.

Reply:
That would be good idea to have one proper method to remove file row from DB on the basis of its Md5Hash value.